### PR TITLE
add tools to check ASF headers in code files.

### PR DESCRIPTION
**Pull request by Firstsawyou**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
add tools to check ASF headers in code files.

### CODE REVIEW

File: .github/workflows/api_ci.yml [hunk 1]
```diff
@@ -74,3 +74,7 @@ jobs:
       working-directory: ./api
       run: |
         go test ./...
+    
+    - name: run Makefile
+      run: |
+        sudo make license-check
```
Comment (moonming) [inline .github/workflows/api_ci.yml:80]: why add `sudo`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+dist: bionic
```
Comment (moonming) [inline .travis.yml:1]: we using github actions, not travis as CI

File: .travis/ASF-Release.cfg [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+grpc_server_example
+.travis.yml
+grpcurl
+t/servroot
+
+conf
+.travis/openwhisk-utilities
+
+# Exclude dashboard files
+dashboard
```
Comment (moonming) [inline .travis/ASF-Release.cfg:95]: this repo not has `dashborad` dir

File: .travis/ASF-Release.cfg [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+# Exclude travis CI files
+build-cache
+deps
+test-nginx
+grpc_server_example
+.travis.yml
+grpcurl
+t/servroot
+
```
Comment (moonming) [inline .travis/ASF-Release.cfg:90]: we don't have those dirs

**Review by moonming — CHANGES_REQUESTED** [review]

File: .travis/ASF-Release.cfg [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+# for test case files of test::nginx
+*.t=has_block_license, eol_at_eof, regex_check
+*.pm=has_block_license,  eol_at_eof, regex_check
+
+# for APISIX CLI
+*/bin/apisix=has_block_license,  eol_at_eof, regex_check
+
+# for luarocks spec files
+*.rockspec=has_block_license,  eol_at_eof, regex_check
+
```
Comment (moonming) [inline .travis/ASF-Release.cfg:48]: please check all lines in this file, only keep what you need.

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,8 @@
+### license-check:    Check apisix-dashboard source code for Apache License
+.PHONY: license-check
+license-check:
+ifeq ("$(wildcard .travis/openwhisk-utilities/scancode/scanCode.py)", "")
+	git clone https://github.com/apache/openwhisk-utilities.git .travis/openwhisk-utilities
+	cp .travis/ASF* .travis/openwhisk-utilities/scancode/
```
Comment (moonming) [inline Makefile:6]: please take a better name, this PR is nothing with travis.

**Comment by juzhiyuan** [comment]
@Firstsawyou This PR is only for the manager-api? or work both for Dashboard and manager API?

**Review by Firstsawyou — COMMENTED** [review]
> @Firstsawyou This PR is only for the manager-api? or work both for Dashboard and manager API?
 work both for Dashboard and manager API.

File: .actions/ASF-Release.cfg [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+*.md=no_tabs, has_block_license, eol_at_eof, regex_check
+*.properties=no_tabs, has_block_license,  eol_at_eof, regex_check
+*.py=no_tabs, has_block_license,  eol_at_eof, regex_check
+*.scala=has_block_license, no_tabs,  eol_at_eof, regex_check
+*.sh=has_block_license,  eol_at_eof, regex_check
+*.swift=no_tabs, has_block_license,  eol_at_eof, regex_check
+*.yaml=has_block_license,  eol_at_eof, regex_check
+*.yml=has_block_license,  eol_at_eof, regex_check
+*.css=has_block_license, regex_check
+*Dockerfile*=has_block_license
```
Comment (moonming) [inline .actions/ASF-Release.cfg:37]: we need add rule for `.tsx` files.

**Review by moonming — CHANGES_REQUESTED** [review]

**Comment by moonming** [comment]
@Firstsawyou please check the output of CI.
you need to fix all error messages.

File: .actions/ASF-Release.cfg (hunk 7 above)
Comment (Firstsawyou) [reply .actions/ASF-Release.cfg:37]: ok.

**Comment by Firstsawyou** [comment]
> @Firstsawyou please check the output of CI: https://github.com/apache/apisix-dashboard/pull/454/checks?check_run_id=1085838821
> you need to fix all error messages.
okay.

**Review by moonming — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

File: .actions/ASF-Release.cfg [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+src/components/NoticeIcon
+src/components/PageLoading/index.tsx
+src/components/RightContent
+src/e2e/__mocks__/antd-pro-merge-less.js
+src/e2e/baseLayout.e2e.js
+src/pages/404.tsx
+src/service-worker.js
+compose/dashboard_conf/nginx.conf
+docker/nginx.conf
+
```
Comment (moonming) [inline .actions/ASF-Release.cfg:81]: where these files come from? we ONLY check code files in repo, not include dependent files.
you should skip dir of dependent files.

**Pull request merged**